bitrise.yml

```diff
@@ -74,6 +74,11 @@ workflows:
             "$BITRISE_SOURCE_DIR/Tests/UnitTest.xctestplan" \
             "$BITRISE_SOURCE_DIR/Client/" \
             "$BITRISE_SOURCE_DIR/Tests/XCUITests"
+
+            echo $BITRISE_SOURCE_DIR
+            ls $BITRISE_SOURCE_DIR
+            echo $BITRISE_XCRESULT_PATH
+            ls $BITRISE_XCRESULT_PATH
```

Comment:
This is where the .xcresult file is stored. This file is in `~/DerivedData/Client-xxxx/Logs/Test/Test-Fennec-2023.07.28_13-25-45-+0200.xcresult`, with that format, Test-SchemeName-Date+number.xcresult